docs/src/pages/theming.mdx

```diff
@@ -0,0 +1,430 @@
+import { Callout } from "nextra-theme-docs";
+
+# Theming
+
+## Components anatomy
+
+### UploadButton
+
+Simplified component structure:
+
+```jsx
+<div data-state={getUtState()} className={className}>
```

Comment:
What is `getUtState`? Not something users should care about

Reply:
Then probably makes sense to replace it with inline comment saying something like `current state` or `readying, ready or uploading`(to specify values that can be here)